settings/settings.go

```diff
@@ -227,3 +276,312 @@ func objectValidator(bound int, args ...string) bool {
+				if err != nil {
+					fmt.Println(err)
+					fmt.Printf("error adding connection %s to file\n", varSplit[0])
+				}
+			}
+		}
+		fmt.Println("Aiflow connections successfully export to the file " + envFile + "\n")
+		rmCmd := "rm tmp.connection"
+		_ = execAirflowCommand(id, rmCmd)
+		return nil
+	}
+	return errors.New("connection export unsuccessful")
+}
+
+func Export(id, settingsFile string, version uint64, connections, variables, pools, logs bool) error {
+	if id == "" {
+		return errNoID
+	}
+	// init settings file
+	err := InitSettings(settingsFile)
+	if err != nil {
+		return err
+	}
+	var parseErr bool
+	// export Airflow Objects
+	if version >= AirflowVersionTwo {
+		if pools {
+			err = ExportPools(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if variables {
+			err = ExportVariables(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if connections {
+			err := ExportConnections(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if parseErr {
+			return errors.New("there was an error during export")
+		}
+		return nil
+	}
+
+	return errors.New("Command must be used with Airflow 2.X")
+}
+
+func ExportConnections(id string, logs bool) error {
+	// Setup airflow command to export connections
+	airflowCommand := "airflow connections list -o yaml"
+	out := execAirflowCommand(id, airflowCommand)
+	if logs {
+		fmt.Println("Export Connections logs:\n" + out)
+	}
+	// remove all color from output of the airflow command
+	plainOut := re.ReplaceAllString(out, "")
+	// remove extra warning text
+	yamlCons := "- conn_id:" + strings.SplitN(plainOut, "- conn_id:", 2)[1]
+
+	var connections ListConnections
+
+	err := yaml.Unmarshal([]byte(yamlCons), &connections)
+	if err != nil {
+		return err
+	}
+	// add connections to settings file
+	for i := range connections {
+		port, err := strconv.Atoi(connections[i].ConnPort)
+		if err != nil {
+			fmt.Println("Issue with parsing port number: ")
+			fmt.Println(err)
+		}
+		fmt.Println("Exporting Connection: " + connections[i].ConnID)
+
+		newConnection := Connection{
+			Conn_ID:       connections[i].ConnID,
+			Conn_Type:     connections[i].ConnType,
+			Conn_Host:     connections[i].ConnHost,
+			Conn_Schema:   connections[i].ConnSchema,
+			Conn_Login:    connections[i].ConnLogin,
+			Conn_Password: connections[i].ConnPassword,
+			Conn_Port:     port,
+			Conn_URI:      connections[i].ConnURI,
+			Conn_Extra:    connections[i].ConnExtra,
+		}
+
+		settings.Airflow.Connections = append(settings.Airflow.Connections, newConnection)
+	}
+	// write to settings file
+	viperSettings.Set("airflow", settings.Airflow)
+	err = viperSettings.WriteConfig()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("successfully exported Connections\n\n")
+	return nil
+}
+
+func ExportVariables(id string, logs bool) error {
```

Comment:
can we combine the common logic of this method &  `EnvExportVariables`, because most of it is the exact same, except where the output is stored